src/code/middleware/its-log.js

```diff
@@ -162,7 +163,7 @@ function createLogEntry(loggingMetadata, action, nextState){
   delete context.type;
   delete context.meta;
 
-  context = makeMutable(context);
+  context = makeMutable(cloneDeep(context));
```

Comment:
@kswenson @sfentress Before I made this change, I was finding the drakes in my `drakes` array had their `sex` set to `male`/`female` instead of `0`/`1`. Copying the context here instead of modifying it directly fixed the problem. It's not clear to me why this only started happening when the authoring started loading from Firebase, am I missing any obvious reasons?